bookwyrm/templates/ostatus/template.html

```diff
@@ -0,0 +1,60 @@
+{% load layout %}
+{% load i18n %}
+{% load static %}
+{% load utilities %}
+{% load markdown %}
+
+<!DOCTYPE html>
+<html lang="{% get_lang %}">
+<head>
+    <title>{% block title %}{% endblock %}{{ site.name }}</title>
```

Comment:
Accessibility-wise, it would be great to have the title set with the current step of the follow